_posts/en/newsletters/2021-02-17-newsletter.md

```diff
@@ -0,0 +1,311 @@
+---
+title: 'Bitcoin Optech Newsletter #136'
+permalink: /en/newsletters/2021/02/17/
+name: 2021-02-17-newsletter
+slug: 2021-02-17-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter mentions a new constant-time algorithm for
+generating and verifying transaction signatures, describes a proposal to
+stop processing unsolicited transactions, summarizes a proposed BIP for
+setting up multisig wallets, shares insight from a discussion about
+managing escrows over LN, links to renewed discussion about
+bidirectional upfront LN fees, and mentions a new protocol for
+mitigating the risk of malicious hardware wallets.  Also included are
+our regular sections with news about updated clients and services,
+announcements of new software releases and release candidates, and
+descriptions of notable changes in popular Bitcoin infrastructure
+software.
+
+## News
+
+- **Faster signature operations:** Russell O'Connor and Andrew Poelstra
+  published a [blog post][safegcd bounds] announcing the implementation
+  of an algorithm that may speed up Bitcoin Core's signature
+  verification by 15%.  It can also reduce by 25% the amount of time it
```

Comment:
"may speed up" and "can reduce" indicate different levels of certainty to me, perhaps "speeds up" and "reduces", or "may speed up" and "may reduce"